frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/EditTab.tsx

```diff
@@ -1,112 +1,69 @@
+          ref={dialogRef}
           component={component}
           handleComponentChange={handleComponentChange}
         />
       ) : (
-        <div className={classes.optionButtons}>
-          <AddManualOptionsModal
-            setComponentHasOptionList={setComponentHasOptionList}
-            component={component}
-            handleComponentChange={handleComponentChange}
-          />
-          <OptionListSelector
-            setComponentHasOptionList={setComponentHasOptionList}
-            component={component}
-            handleComponentChange={handleComponentChange}
-          />
-          <OptionListUploader
-            setComponentHasOptionList={setComponentHasOptionList}
-            component={component}
-            handleComponentChange={handleComponentChange}
-          />
+        <div className={classes.container}>
+          <AddOptionList component={component} handleComponentChange={handleComponentChange} />
         </div>
       )}
-      {errorMessage && (
-        <StudioErrorMessage className={classes.errorMessage} size='small'>
-          {errorMessage}
-        </StudioErrorMessage>
-      )}
+      {errorMessage && <StudioErrorMessage size='small'>{errorMessage}</StudioErrorMessage>}
     </>
   );
 }
 
-type SelectedOptionListProps = {
-  setComponentHasOptionList: (value: boolean) => void;
-} & Pick<IGenericEditComponent<SelectionComponentType>, 'component' | 'handleComponentChange'>;
+type AddOptionListProps = EditTabProps;
 
-function SelectedOptionList({
-  setComponentHasOptionList,
-  component,
-  handleComponentChange,
-}: SelectedOptionListProps) {
+function AddOptionList({ component, handleComponentChange }: AddOptionListProps) {
   const { t } = useTranslation();
 
-  const handleDelete = () => {
-    if (component.options) {
-      delete component.options;
-    }
-
-    const emptyOptionsId = '';
-    handleComponentChange({
-      ...component,
-      optionsId: emptyOptionsId,
-    });
-
-    setComponentHasOptionList(false);
+  const handleInitialManualOptionsChange = () => {
+    handleOptionsChange({ component, handleComponentChange, options: [] });
   };
 
-  const label =
-    component.optionsId !== '' && component.optionsId !== undefined
-      ? component.optionsId
-      : t('ux_editor.modal_properties_code_list_custom_list');
-
   return (
-    <div aria-label={label} className={classes.chosenOptionContainer}>
-      <OptionListEditor
-        label={label}
-        optionsId={component.optionsId}
-        component={component}
-        handleComponentChange={handleComponentChange}
-      />
-      <div className={classes.deleteButtonContainer}>
-        <StudioDeleteButton
-          className={classes.deleteButton}
-          onDelete={handleDelete}
-          title={t('ux_editor.options.option_remove_text')}
-          variant={'tertiary'}
-        />
-      </div>
+    <div className={classes.container}>
+      <StudioButton
+        className={classes.createNewButton}
```

Comment:
Instead of adding css for each of these elements that adjusts their width, why dont set a `width: 60%` fex on the container?

Reply:
Does adding the following sound good?
```css
  width: 60%;
  min-width: 12rem;
  white-space: nowrap;
```
It will result in looking like the image below when at the minimum size of the ResizableLayout.

![image](https://github.com/user-attachments/assets/5d337a02-3f85-4a08-81b0-d87b85624b81)

Or like this normally

![image](https://github.com/user-attachments/assets/9cb38104-4cf7-496c-9441-3f8539c7d908)